project/Dependencies.scala

```diff
@@ -20,6 +20,7 @@ object Dependencies {
 
   val excludeBouncyCastle =     ExclusionRule(organization = "org.bouncycastle", name = s"bcprov-jdk15on")
   val excludeProtobufJavalite = ExclusionRule(organization = "com.google.protobuf", name = "protobuf-javalite")
+  val excludeScalaCache =       ExclusionRule(organization = "com.github.cb372")
```

Comment:
it looks like we're excluding "com.github.cb372" group artifacts?  If so, why?  If not what are we doing with this line along with lines 100 and 101?

Reply:
> this adds an exclusion rule for scalacache to the google2 import. We use scalacache internal to Rawls - though it's a different version in Rawls, and we have compile problems when using the version included in google2. Google2's usage of scalacache appears completely limited to test (https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/test/scala/org/broadinstitute/dsde/workbench/google2/KubernetesManualTest.scala), so it's safe to ignore.